### PR TITLE
Fix trailing comma in comments for object expression (#538)

### PR DESCRIPTION
**Pull request by yamafaktory**

### CODE REVIEW

**Comment by yamafaktory** [comment]
@vjeux This one should fix #538, but only what @amasad initially mentioned. The second issue mentioned by @heyimalex seems to be a different one.

One last thing I was unsure about was how to add a test for that specific case (is the `--trailing-comma` flag used during the tests?) ❓

**Comment by vjeux** [comment]
In the `jsfmt.spec.js`, you can pass a second argument with `{trailingComma: true}`

**Comment by yamafaktory** [comment]
@vjeux I'll do it. Regarding the other issue in arrays, I've found what's causing it https://github.com/jlongster/prettier/blob/master/src/printer.js#L731. Should we create a new issue out of @heyimalex comment for that?

File: src/printer.js [hunk 1]
```diff
@@ -1801,14 +1801,20 @@ function printArgumentsList(path, options, print) {
     ]);
   }
 
+  // For next group, there are some exceptions where we don't want
+  // a trailing comma in the last comment.
+  const noTrailingCommaInLastComment = lastArg.type === "ObjectExpression";
```
Comment (vjeux) [inline src/printer.js]: The issue is that we print the trailing comment as part of the element itself but we put the trailing comma outside.

What is happening:

```
[element
// comment],
```

What we want to happen:

```js
[element],
// comment
```

I'm not really sure how to fix this issue but I doubt that your pull request is the right approach. The issue is that the last argument has a comment, not that it's an ObjectExpression.

Comment (yamafaktory) [inline src/printer.js]: Oh you're right... I was focused on that one https://github.com/jlongster/prettier/issues/538#issue-204423255 but actually this one makes it clear for me now https://github.com/jlongster/prettier/issues/538#issuecomment-276522889.

Comment (yamafaktory) [inline src/printer.js]: @vjeux if you don't mind, the best thing will be to close that PR as this is definitely not the correct approach to solving the problem 👍.

**Pull request closed without merging**